.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+  - "3.6-dev"
+  - "3.7-dev"
+  - "nightly"
+script: make GREP=/bin/grep LN=/bin/ln MKDIR=/bin/mkdir PYTHON_VERSION=3.6.0 SH=/bin/sh TAR=/bin/tar test
```

Comment:
Since this hasn't been merged yet, I think it would be cool to [amend](https://help.github.com/articles/changing-a-commit-message/) your commit to remove things that aren't intended to be part of it, and then do a force push on your l0b0:master branch to update this PR.